helpers/mailer.js

```diff
@@ -0,0 +1,73 @@
+const nodemailer = require('nodemailer');
+const sgMail = require('@sendgrid/mail');
+const getLogger = require('./get-logger');
+const configEnv = require('../config.env');
+
+class Mailer {
+  constructor() {
+    this.logger = getLogger('mailer');
+    //nodemailer direct sender
+    // this._transport = nodemailer.createTransport(configEnv.mail);
+
+    //smtp nodemailer + sendgrid
```

Comment:
if you use nodemailer you should remove sendgrid and sgMail, SGKey ...  from all places